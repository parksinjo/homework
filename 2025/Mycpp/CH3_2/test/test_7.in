1
7
GilDong 1 70.5 true :Jongno-gu, Seoul:
0
0
