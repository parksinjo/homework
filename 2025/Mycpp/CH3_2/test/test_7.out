
******************************* Main Menu *********************************
* 0.Exit 1.CurrentUser(ch3_2)                                             *
* 2.Class:Object(ch3_1)                                                   *
***************************************************************************
Menu item number? 1
Person::Person(...):p0 0 70 false :Gwangju Nam-gu Bongseon-dong 21:

+++++++++++++++++++++ Current User Menu ++++++++++++++++++++++++
+ 0.Logout 1.Display 2.Getter 3.Setter 4.Set 5.WhatAreYouDoing +
+ 6.IsSame 7.InputPerson                                       +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Menu item number? 7
input person information:
GilDong 1 70.5 true :Jongno-gu, Seoul:
GilDong 1 70.5 true :Jongno-gu, Seoul:

+++++++++++++++++++++ Current User Menu ++++++++++++++++++++++++
+ 0.Logout 1.Display 2.Getter 3.Setter 4.Set 5.WhatAreYouDoing +
+ 6.IsSame 7.InputPerson                                       +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Menu item number? 0
Person::~Person():GilDong 1 70.5 true :Jongno-gu, Seoul:
Person::~Person():p0 0 70 false :Gwangju Nam-gu Bongseon-dong 21:
Person::~Person():p0 0 70 false :Gwangju Nam-gu Bongseon-dong 21:

******************************* Main Menu *********************************
* 0.Exit 1.CurrentUser(ch3_2)                                             *
* 2.Class:Object(ch3_1)                                                   *
***************************************************************************
Menu item number? 0
Good bye!!
